.github/workflows/main.yml

```
@@ -13,16 +13,16 @@ jobs:
 
     steps:
       - name: Git checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Use cache
-        uses: actions/cache@v3
+        uses: actions/cache@v4
         with:
           path: ~/.m2
           key: "${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}"
           restore-keys: |
             ${{ runner.os }}-maven3-
       - name: Configure Java
-        uses: actions/setup-java@v3
+        uses: actions/setup-java@v4
         with:
           java-version: "8"
```

```suggestion
          java-version: "11"
```

Seems like due to a newer MacOS version on GitHub Actions, there's no Java 8, so probably time to bump to Java 11 then.